src/facets/common/NiftyApesInternal.sol

```diff
@@ -1,12 +1,12 @@
 //SPDX-License-Identifier: MIT
-pragma solidity 0.8.18;
+pragma solidity 0.8.21;
 
 import "@openzeppelin/contracts/security/PausableUpgradeable.sol";
 import "@openzeppelin/contracts/security/ReentrancyGuardUpgradeable.sol";
-import "@openzeppelin/contracts/token/ERC721/IERC721Upgradeable.sol";
+// import "@openzeppelin/contracts/token/ERC721/IERC721Upgradeable.sol";
 import "@openzeppelin/contracts/token/ERC721/extensions/ERC721URIStorageUpgradeable.sol";
 import "@openzeppelin/contracts/token/ERC721/utils/ERC721HolderUpgradeable.sol";
-import "@openzeppelin/contracts/token/ERC721/extensions/IERC721MetadataUpgradeable.sol";
+// import "@openzeppelin/contracts/token/ERC721/extensions/IERC721MetadataUpgradeable.sol";
```

Comment:
do we need these commented out import statements?

Reply:
nope, removed